components/select_team/select_team.jsx

```diff
@@ -215,9 +250,16 @@ export default class SelectTeam extends React.Component {
                             defaultMessage='Teams you can join: '
                         />
                     </h4>
-                    <div className='signup-team-all'>
+                    <InfiniteScroll
+                        callBack={this.fetchMoreTeams}
+                        styleClass='signup-team-all'
+                        totalItems={totalTeamsCount}
+                        itemsPerPage={TEAMS_PER_PAGE}
+                        pageNumber={currentPage}
+                        loaderStyle={{padding: '0px'}}
```

Comment:
Can you add a height of 40px to this without that loader is not visible all that much.
![Oct-14-2019 17-02-41](https://user-images.githubusercontent.com/4973621/66748489-ee34dd80-eea4-11e9-8543-c1f0159726ab.gif)

Reply:
Hello @sudheerDev , I am sorry I don't really understand this. Should I add the height of 40px to the loader component or the infinite scroll?

Reply:
loader component as a default value so it has some height for the loader

Reply:
Hi @sudheerDev , worked on this.